modules/Charts/Dashlets/ModuleActivityDashlet/ModuleActivityConfigure.tpl

```diff
@@ -0,0 +1,103 @@
+{*
```

Comment:
Please update license in the file. See https://suitecrm.com/wiki/index.php/Coding_Standards for details.